extra/modules/pb-richmedia-filter/src/main/java/org/prebid/server/hooks/modules/pb/richmedia/filter/core/BidResponsesMraidFilter.java

```diff
@@ -0,0 +1,63 @@
+package org.prebid.server.hooks.modules.pb.richmedia.filter.core;
+
+import com.iab.openrtb.response.Bid;
+import org.apache.commons.lang3.StringUtils;
+import org.prebid.server.auction.model.BidderResponse;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderError;
+import org.prebid.server.bidder.model.BidderSeatBid;
+import org.prebid.server.hooks.modules.pb.richmedia.filter.model.AnalyticsResult;
+import org.prebid.server.hooks.modules.pb.richmedia.filter.model.MraidFilterResult;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class BidResponsesMraidFilter {
+
+    private static final String TAG_STATUS = "success-block";
+    private static final Map<String, Object> TAG_VALUES = Map.of("richmedia-format", "mraid");
+
+    private final String mraidScriptPattern;
+
+    public BidResponsesMraidFilter(String mraidScriptPattern) {
+        this.mraidScriptPattern = mraidScriptPattern;
```

Comment:
Please, add call to `Objects.reqireNonNull`, so that creation will fail in case we have provided null string.